www/heatmap.html

```diff
@@ -0,0 +1,239 @@
+            <p><br>Calling the <code class="javascript">setData()</code> function on the <code class="javascript">heatmap</code> variable: <code class="javascript">heatmap.setData(data)</code> allows the user to provide the heatmap instance with data to display. The data has to be in JSON format, below is an example of how the information should be structured.</p>
+
+<pre><code class="javascript hljs">{
+    max: 5, // A maximum value is optional
+    min: 0, // A minimum value is optional
+    data: [{ x: 10, y: 15, value: 5}, ...] // Note that the data paramater is an array of json objects with x, y, and value variables
+}</code></pre>
+
+            If your data is stored as a CSV File, start by converting your CSV file into a JSON Object, a quick online search of "CSV to JSON" will return sites that can help with this. Once you have a <code class="javascript">.json</code> file, check that the embedded JSON array has a key value of <code class="javascript">data</code> like in the JSON Object structure above. The script given later operates under the assumption that the array is referred to as <code class="javascript">data</code>
+
+            <p><br>Edit your <code class="javascript">.json</code> file by inserting <code class="javascript">var heatmap_data =</code> infront of your JSON Object and then replace the <code class="javascript">.json</code> extension with <code class="javascript">.js</code>. Thus <code class="javascript">heatmap-data.json</code> would appear in explorer as <code class="javascript">heatmap-data.js</code>.</p>
+
+<pre><code class="javascript hljs">var heatmap_data = {
+    max: 5, // A maximum value is optional
+    min: 0, // A minimum value is optional
+    data: [{ x: 10, y: 15, value: 5}, ...] // Note that the data paramater is an array of json objects with x, y, and value variables
+}</code></pre>
+
+            <p><br>Import the <code class="javascript">heatmap-data.js</code> file into your HTML page, similarly to the second step of this guide.</p>
+
+<pre><code class="html">&lt;!-- Script Element --&gt;
+&lt;script src="./js/heatmap.js"&gt;&lt;script&gt; 
+&lt;script src="./js/heatmap-data.js"&gt;&lt;script&gt; // the file we just created is now imported
+</code></pre>
+
+            <p>Importing the <code class="javascript">heatmap-data.js</code> gives the heatmap instance access to the information to be displayed. <br><br>The script below displays the data sequentially at a constant rate and can be downloaded at the end of this page. The different variables in the script can be set to different timings: <code class="javascript">initial_delay</code> is the amount of time after the webpage finishes loading before starting the visualization and <code class="javascript">heatmap_display_delay</code> is the amount of time between each additional point being displayed. <b>Note:</b> All times are represented in milliseconds.</p>
```

Comment:
It might be easier for the user if you replaced "... and can be downloaded at the end of this page" with "... and can be downloaded `<a href="/js/heatmap-data.js" download>here</a> ` "